src/scs/pki.md

```diff
@@ -0,0 +1,31 @@
+# Public Key Infrastructure
+
+In SCS context a public key infrastructure (PKI) term refers to a system for creation, storage, and distribution of digital certificates which are used to verify that a particular key belongs to a certain entity. PKI is creating and managing a hierarchical set of digital certificates which map public keys to certain entities. Storage and revocation methodologies are to be defined.
+
+The PK of SCS should consist of:
+  + A certificate authority (CA) responsible for storing, issuing and signing the digital certificates
+  + A registration authority (RA) responsible for requesting entity identity verification
+  + A central directory responsible for secure storage of the keys
+  + A Certificate Management System (CMS) managing access to stored certificates
+ 
+## Private Certificate Authority (PCA)
+
+PCA enables creation of private certificate authority hierarchies, consisting of root and subordinate CAs. It issues end-entity X.509 certificates, that are used for: 
+
+  + Encrypted TLS communication channels (data encryption in transit)
+  + Code and image signing
+
+PCA can be established in the cloud or on premise. Initially, openssl based on-prem solution is assumed, however, some of the target projects might consider using commercial cloud solutions.
```

Comment:
Let's think together on "...openssl based on-prem solution is assumed...".
Considering that on premise -> on-premises and openssl -> OpenSSL, what do we have?
... the OpenSSL solution deployed on-premises is assumed... -- if we write like this, the meaning is not distorted?

Reply:
"Initially, the OpenSSL based solution deployed on-premises is assumed" will not change the meaning ... But this sentence might soon change as I will add more text on HSM as per comment from Brian.

Reply:
As we know, there is nothing more permanent than temporary)
Let's use the updated version until the next iteration.